modules/installing-gcp-user-defined-labels-and-tags.adoc

```diff
@@ -49,25 +42,26 @@ User-defined labels and {product-title} labels are available on the following GC
 
 .User-defined tags
 
-User-defined tags are attached to resources created by the {product-title} Image Registry Operator and not on the resources created by any other Operators or the Kubernetes in-tree components.
+User-defined tags are applied only to resources created by OpenShift Container Platform installation program and its core components such as:
+* GCP filestore CSI Driver Operator
```

Comment:
In rendered doc, these does not appear as bullet points.

Reply:
Done